### DISCUSSION TITLE
How to setup hawkeye superset publish button for new report on diksha

### DISCUSSION THREAD

**Discussion by kamal-singhal** [Sunbird-Obsrv/Community#131] [Issues]
Hi team, 

Need your support to tell us how to setup publish button in hawkeye superset to publish new report on portal. 
Please let us know complete pipeline workflow from end to end. 
If there is any APIs for the same please let us share the curl for the same.

**Comment by manjudr** [comment]
@kamal-singhal  - Please go through this developer setup documentation to set the report configurator - https://obsrv.sunbird.org/learn/product-and-developer-guide/report-configurator

Please let us know if you are still encountering any issues

FYI - These are the GitHub repos to set up the report 
1.  https://github.com/Sunbird-Ed/incubator-superset
2. https://github.com/Sunbird-Obsrv/report-configurator
